static-analysis/StaticAnalysisChapter.md

```diff
@@ -447,17 +447,17 @@ b = body(e)
     end
 ~~~
 
-    We look at each expression in the body of `e`. If it is a label, we check to see if there is a `goto` that jumps to this label (and occurs after the current index). If the result of `findnext` is greater than zero, then such a goto node exists, so we'll add that to `loops` (the `Array` of loops we are currently in) and increment our `nesting` level.
+We look at each expression in the body of `e`. If it is a label, we check to see if there is a goto that jumps to this label (and occurs after the current index). If the result of `findnext` is greater than zero, then such a goto node exists, so we'll add that to `loops` (the `Array` of loops we are currently in) and increment our `nesting` level.
```

Comment:
This also should be consistent with the above `goto` expression stuff. I would put `GotoNode` here, but the code is also handling `Expr`s with head `:gotoifnot`, so it wouldn't be totally correct.